src/api/test/functional/attributes_test.rb

```diff
@@ -379,9 +379,29 @@ def test_create_attributes_project
     assert_response :success
     assert_xml_tag(tag: 'revisionlist')
     revision = Xmlhash.parse(@response.body).elements('revision').last
-    assert_equal 'tom', revision['user']
+    assert_equal 'king', revision['user']
     srcmd5 = revision['srcmd5']
 
+    # check history on changed value only
```

Comment:
sorry, but the test is called test_create_attributes_project - testing updates from within is the reason why developers loath the minitests :(

Reply:
well, even if I agree with you, the same test destroy attributes to.. so I guess it's a general issue in the minitest test suite.. handling it as  nitpick.